plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_launcher_util.py

```diff
@@ -46,13 +46,13 @@ def _run_job(
 
 
 def launch_job_on_ray(
-    ray_remote_cfg: DictConfig,
+    ray_remote: DictConfig,
     sweep_config: DictConfig,
     task_function: TaskFunction,
     singleton_state: Any,
 ) -> Any:
-    if ray_remote_cfg:
-        run_job_ray = ray.remote(**ray_remote_cfg)(_run_job)
+    if ray_remote:
+        run_job_ray = ray.remote(**ray_remote)(_run_job)
     else:
```

Comment:
seeing this makes me wonder if the config should look like:
```yaml
hydra:
  launcher:
    ray:
      init:
        ..
      remote:
        ..
 ```

This seems to model things better. and if we ever want to configure another Ray function we won't need to repeat the ray_ prefix again everywhere.

Reply:
that's a great idea!

I'm actually working on adding a new node (`run_env`) which would also need the prefix probably if we go with the current way. 

will modify the configs.

Reply:
awesome :)